src/showcase/ui-group/ButtonCards.tsx

```diff
@@ -71,6 +72,12 @@ const data = [
     link: '/components/button/shine-button',
     component: <ShineButton />,
   },
+  {
+    id: 11,
+    title: 'Text Revel',
```

Comment:
I am still seeing `Text Revel` for the `title`.

Reply:
sorry ,Im so dump, fixed